tests/integration/Cargo.toml

```diff
@@ -31,6 +31,7 @@ midenc-codegen-masm.workspace = true
 midenc-session.workspace = true
 midenc-compile.workspace = true
 midenc-debug.workspace = true
+cargo-miden = { path = "../../tools/cargo-miden" }
```

Comment:
*Suggestion:* We may as well add `cargo-miden` to the workspace `Cargo.toml`, since it's going to get picked up as a dependency at the workspace level anyway due to its presence here.

Reply:
Done.